src/main/java/com/softserve/edu/sporthubujp/controller/ArticleController.java

```diff
@@ -29,7 +36,14 @@ public ResponseEntity<ArticleDTO> getArticleById(@PathVariable String id) {
         return ResponseEntity.status(HttpStatus.OK).body(
             articleService.getArticleById(id));
     }
+    @GetMapping("/{id}")
```

Comment:
It will conflict with the previous endpoint. You can change to 

@GetMapping("/{id}/comments")